packages/config/config/devices/0x0179/ch-103.json

```diff
@@ -0,0 +1,176 @@
+	"manufacturerId": "0x0179",
+	"label": "CH-103",
+	"description": "CH-103 Relay",
+	"devices": [
+		{
+			"productType": "0x0041",
+			"productId": "0x0014"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Target for temperature and alarm reports",
+			"maxNodes": 10,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Automatic Calibration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal",
+					"value": 0
+				},
+				{
+					"label": "Calibration",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Standard",
+					"value": 0
+				},
+				{
+					"label": "Motor with limit switchs",
+					"value": 1
+				},
+				{
+					"label": "Blind Control",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Rotation Time",
+			"description": "Time of full rotation of lamellas or (180 °) in the blind mode",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "4",
+			"label": "Open Operating Time",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Close Operating Time",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "6",
+			"label": "Open Delay",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "7",
+			"label": "Close Delay",
+			"unit": "decisecond",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 0
+		},
+		{
+			"#": "50",
+			"label": "Response Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Security and regular commands",
+					"value": 0
+				},
+				{
+					"label": "Security commands only",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "51",
+			"label": "2nd Association Group",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Click",
+					"value": 0
+				},
+				{
+					"label": "Press",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "52",
+			"label": "3rd Association Group",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Click",
+					"value": 0
+				},
+				{
+					"label": "Press",
+					"value": 1
+				}
+			]
+		}
```

Comment:
I don't understand these either and the `associations` property above only lists a single association group.

Reply:
Corrected by adding the associations groups.

Reply:
Rereading descriptions I think it is actually 0 is "usual" and 1 is "secure" for transmits.  Not sure the best words to use for that. 

Translation: `0 - When you click on Switch 1, send the usual command to the 2nd group of links (Default);; 1 - When you press switch 1, send a secure command to the 2nd group of connections.`

Reply:
You can use this template we've been using in other devices too:
```jsonc
{
		"$import": "~/templates/master_template.json#base_enable_disable",
		"label": "Send Secure Commands to 2nd Association Group",
		"description": "Only active when the node is included securely"
	}
```
The default is 0, so this should work as-is